geoserver-init/010_create-layers.js

```diff
@@ -0,0 +1,47 @@
+import GeoServerRestClient from 'geoserver-node-client';
```

Comment:
nitpick: file is named `create-layers` but it just creates workspaces, or am i wrong?

Reply:
good catch!